packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -388,38 +401,40 @@ pc.rollBackLocalDescription = (peerConnection) => {
 
 /**
  * @param {Object} params {
- *  offerToReceiveAudio: {Boolean} sdp constraints
- *  offerToReceiveVideo: {Boolean} sdp constraints
- *  offerSdp: {sdp}
- *  stream: {pcStream}
- * }
- * @param {Object} meetingId meetingId for sending metrics 
- * @returns {Array} [MediaSDP, ScreenSDP]
+ * @param {Boolean} params.offerToReceiveAudio
+ * @param {Boolean} params.offerToReceiveVideo
+ * @param {string} params.offerSdp
+ * @param {MediaStream} params.stream
+ * @param {Object} meetingProperties
+ * @param {string} meetingProperties.meetingId
+ * @param {string} meetingProperties.remoteQualityLevel LOW|MEDIUM|HIGH
```

Comment:
For situations like these, if we have a Record/Array of constants that could be used in a method, what do we prefer for documenting this?

I'd think:
```js
/**
 * @param {string} meetingProperties.remoteQualityLevel[LOW|MEDIUM|HIGH]
 */
```
or
```js
/**
 * @param {string} ...meetingProperties.remoteQualityLevel
 */
```

Reply:
I actually think a [@typedef](https://jsdoc.app/tags-typedef.html) would be good for this.